src/UglyToad.PdfPig.DocumentLayoutAnalysis/Export/PageXmlGeneralExporter.cs

```diff
@@ -0,0 +1,306 @@
+﻿namespace UglyToad.PdfPig.DocumentLayoutAnalysis.Export
```

Comment:
This is a copy of `PageXmlTextExporter` with changes to make it more general. With this you pass a list of IBlocks and it generates the PageXml for that.

Reply:
Will Raise a separate PR and can discuss on that